src/lib/utils/application-commands/ApplicationCommandRegistries.ts

```diff
@@ -53,6 +53,7 @@ export async function handleRegistryAPICalls() {
 
 	const { applicationCommands, globalCommands, guildCommands } = await getNeededRegistryParameters();
 
+	// TODO(vladfrangu): v3/v4 see if this can benefit from Promise.all instead
```

Comment:
TODO

Reply:
As stated on discord, if you're not going to pick this up in this PR then greater a GH issue out of it and remove the to-do comment.

I don't want to see these kind of Todo comments in the code

- it's not clear for contributors
- they'll be forgotten
  - you stated that's the intent, in that case don't add them and don't create the issue.
- Todo comments in code are an anti-pattern. They're effectively useless.